app/src/main/java/org/oppia/android/app/databinding/ImageViewBindingAdapters.java

```diff
@@ -150,7 +149,7 @@ public static void setImageDrawable(
    * Binding adapter for profile images. Used to either display a local image or custom
    * colored avatar.
    *
-   * @param imageView view where the profile avatar will be loaded into
+   * @param imageView     view where the profile avatar will be loaded into
```

Comment:
One space is correct actually--please fix.

Reply:
Done.